configure.ac

```diff
@@ -817,8 +818,27 @@ if test "x$included_pcre2" != "xyes"; then
     # and so AC_CHECK_LIB won't work (can't use a variable as library name)
     # AC_SEARCH_LIBS will use the existing $LIBS flags with no additional library first
     AC_SEARCH_LIBS([pcre2_compile_$WCHAR_T_BITS], [],
-      [ working_pcre2=yes
-        AC_MSG_NOTICE([using system PCRE2 library])
+      [ # pcre2 lib found, check for minimum version
+        AC_MSG_CHECKING([for PCRE2 $MIN_PCRE2_VERSION or later])
+        PCRE2_VERSION=`$PCRE2_CONFIG --version`
+        AX_COMPARE_VERSION([$PCRE2_VERSION], [ge], [$MIN_PCRE2_VERSION],
```

Comment:
Any reason not to use `AS_VERSION_COMPARE` which is built in? See the Doxygen check.